static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-design-variations.js

```diff
@@ -0,0 +1,126 @@
+define([
+    'common/modules/commercial/contributions-utilities',
+    'lib/template',
+    'lodash/objects/assign',
+    'raw-loader!common/views/acquisitions-epic-design-variations.html'
+], function(
+    contributionUtilities,
+    template,
+    assign,
+    acquisitionEpicDesignVariations
+) {
+
+    // Building the Epic component using the template acquisitions-epic-design-variations.html
+    // =======================================================================================
+
+    // Default arguments for the template.
+    function defaultTemplateArguments() {
+        return {
+            // copy
+            p1: '… we’ve got a small favour to ask. More people are reading the Guardian than ever, but far fewer are paying for it. Advertising revenues across the media are falling fast. And unlike some other news organisations, we haven’t put up a paywall – we want to keep our journalism open to all. So you can see why we need to ask for your help. The Guardian’s independent, investigative journalism takes a lot of time, money and hard work to produce. But we do it because we believe our perspective matters – because it might well be your perspective, too.',
+            p1Highlight: '',
+            p1PostHighlight: '',
+            p2: 'If everyone who reads our reporting, who likes it, helps to support it, our future would be much more secure.',
+            p3: '',
+
+            // element classes
+            epicClass: '' ,
+            epicTitleClass: '',
+            p1Class: '',
+            p1HighlightClass: '',
+            supporterButtonClass: '',
+            contributorButtonClass: ''
+        };
+    }
+
+    // Merges submitted and default template arguments.
+    function buildTemplateArguments(args) {
+        var out = defaultTemplateArguments();
+        assign(out, args);
+        return out;
+    }
+
+    function buildHtml(args) {
+        return template(acquisitionEpicDesignVariations, buildTemplateArguments(args));
+    }
+
+    // Building a test variant
+    // =======================
+
+    // Common variant properties.
+    var maxViews = {};
+    var successOnView = true;
+
+    function defaultVariantArgs() {
```

Comment:
This can also just be an `Object` literal, and as `maxViews` and `successOnView` are not modified anywhere their values can be declared in the `Object` definition. ie
```
var defaultVariantArgs = {
    maxViews: {},
    successOnView: true
}
```